spec/requests/recipes/recipe_spec.rb

```diff
@@ -195,4 +195,25 @@
       end
     end
   end
+
+  describe 'GET show' do
+    subject { get(recipe_path(recipe.id)) }
+
+    let!(:user) { create(:user) }
+    let!(:recipe) { create(:recipe, user_id: user.id) }
+
+    context 'when logged in' do
+      before { sign_in user }
+
+      it 'have http status 200' do
+        expect(subject).to eq(200)
+      end
```

Comment:
Could we add another expectation to check some recipe properties (name for example)?
You can set manually an attribute when calling the create factory in the let!(:recipe) definition and then add a new it block to check for that property being displayed on show page. Does it make sense ?